clouds/aws/aws_roles/aws_roles.py

```diff
@@ -0,0 +1,66 @@
+import botocore.errorfactory
+from aws.aws_clients import IAMClient
+from botocore.exceptions import BotoCoreError
+from simple_logger.logger import get_logger
+
+LOGGER = get_logger(name=__name__)
+
+
+class AWSRoles:
+    def __init__(self, client: IAMClient):
+        """
+        Manages AWS roles and role-policies.
+
+        Args:
+            client (IAMClient): AWS roles managements expects an IAM client entity
+        """
+        self.client = client
+
+    def create_role_policy(self, role_name, policy_name, policy_json_path):
+        """
+        Create the role policy with specific configuration.
+
+        Args:
+            role_name (str): role policy name
+            policy_name (str): policy name
+            policy_json_path (str): path to json file that holds the policy documents
+
+        Raises:
+            If POST request failed
+        """
+        LOGGER.info(f"Creating new role {role_name} for policy {policy_name}.")
+        if not self.check_available_role_policy(role_name=role_name):
```

Comment:
Don't we want to raise if check_available_role_policy?
if line 32 not match you return None, and the user think all was done.

Reply:
The check method logging in [line 63](https://github.com/RedHatQE/cloud-tools/pull/14/files#diff-773825122a0092f2831ee9d5485dc08a8701fe6a32a2ea7ad33c0dfb2cbbb95dR63) LOGGER.info(f"Policy role {role_name} exists.")